Backpack/Button/Classes/BPKButton.m

```diff
@@ -503,6 +528,12 @@ + (UIColor *)highlightedWhite {
     return [UIColor colorWithRed:0.871 green:0.867 blue:0.878 alpha:1];
 }
 
++ (UIColor *)highlightedOutline {
+    // white overlayed with gray900 at 15%
+    // TODO: Add a method to blend programatically via BPKColor
+    return [BPKColor.white colorWithAlphaComponent:0.8];
```

Comment:
This comment says it's white overlaid with gray at 15%, but the code seems to just be white at 80% opacity?